Source/WebCore/platform/graphics/Image.h

```diff
@@ -135,6 +135,7 @@ class Image : public RefCounted<Image> {
     virtual String accessibilityDescription() const { return String(); } // null string if unknown
 
     virtual void destroyDecodedData(bool /*destroyAll*/ = true) { }
+    virtual void destroyAndCacheDecodedData() { }
```

Comment:
Maybe setDecodedDataVolatile() or markDecodeDataPurgeable()?

Reply:
As above, I was trying to avoid using the terminology "volatile" and "purgeable" since it seemed like these should be WebKit layer concepts.